### PR TITLE
Update docs to reflect that they're in the new repo

### PR DESCRIPTION
**Pull request by blairconrad**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -19,7 +18,7 @@ The analyzer currently provides the following diagnostics:
...
-| FakeItEasy0003 | Argument constraint outside call spec    | no        | Triggered when you try to use an [argument constraint](argument-constraints.md) outside of a [call specification](specifying-a-call-to-configure.md). |
-| FakeItEasy0004 | Argument constraint nullability mismatch | yes       | Triggered when you use a non-nullable [argument constraint](argument-constraints.md) for a nullable parameter. Calls where the argument is null won't be matched. If this is intentional, consider using `A<T?>.That.IsNotNull()` instead. If it's not, make the argument constraint nullable (`A<T?>`). |
-| FakeItEasy0005 | Argument constraint type mismatch        | yes       | Triggered when you use an [argument constraint](argument-constraints.md) whose type doesn't match the parameter. No such calls can be matched.        |
+| FakeItEasy0003 | Argument constraint outside call spec    | no        | Triggered when you try to use an [argument constraint](https://fakeiteasy.readthedocs.io/en/latest/argument-constraints/) outside of a [call specification](https://fakeiteasy.readthedocs.io/en/latest/specifying-a-call-to-configure/). |
```
Comment (blairconrad) [inline docs/index.md]: I'm not yet sure if there's a better way to refer to links in the parent project. Given version drift between the two, maybe this is the best we can do. I think it's worth trying for now.

File: docs/index.md [hunk 2]
```diff
@@ -8,6 +6,7 @@ bugs at runtime.
 
 The analyzer works in Visual Studio 2015 Update 1 or later. Install one of these
 NuGet packages in each project that needs it:
+
```
Comment (blairconrad) [inline docs/index.md]: A bonus change. With this, the list renders as a list. I can revert if you like

File: docs/index.md (hunk 1 above)
Comment (thomaslevesque) [reply docs/index.md]: I'm a little disappointed there's not a mechanism for inter-project links. Well, I guess we can live with it.

File: docs/index.md (hunk 2 above)
Comment (thomaslevesque) [reply docs/index.md]: No, it's fine. Strange we didn't notice this sooner

File: docs/index.md [hunk 3]
```diff
@@ -8,6 +6,7 @@ bugs at runtime.
 
 The analyzer works in Visual Studio 2015 Update 1 or later. Install one of these
```
Comment (thomaslevesque) [inline docs/index.md]: I think that's no longer true... Not sure if we should revert the changes or update the docs.
(I switched the analyzers to .NET Standard during my work to make the build pass on Linux. They were referencing an old version of Roslyn, which caused warnings in tests when targeting .NET Core)

Comment (thomaslevesque) [reply docs/index.md]: Let's merge anyway so that we can see how the docs look. We can fix this later.

**Comment by thomaslevesque** [comment]
Thanks @blairconrad!

**Comment by blairconrad** [comment]
Thanks for the merge!

File: docs/index.md (hunk 3 above)
Comment (blairconrad) [reply docs/index.md]: Yeah, I had some misgivings about that line, but came to the same conclusion you did.
